src/test/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetterTest.java

```diff
@@ -298,4 +298,61 @@ void createGetTxInfoToSignV2_noEventFound_BestBlockFound() {
             btcTransaction
         ));
     }
+
+    @Test
+    void getTxInfoToSign_whenTransactionReceiptNotFoundInSubsequentBlock_shouldThrowHSMRelaseCreationInformationException() {
+        // The event that is searched is not found in the first block
+        Keccak256 blockHash = TestUtils.createHash(3);
+        Keccak256 rskTxHash = TestUtils.createHash(1);
+
+        Transaction transaction = mock(Transaction.class);
+        when(transaction.getHash()).thenReturn(rskTxHash);
+        when(transaction.getReceiveAddress()).thenReturn(PrecompiledContracts.BRIDGE_ADDR);
+
+        TransactionReceipt transactionReceipt = mock(TransactionReceipt.class);
+        when(transactionReceipt.getTransaction()).thenReturn(transaction);
+        when(transactionReceipt.getLogInfoList()).thenReturn(new ArrayList<>());
+
+        TransactionInfo transactionInfo = mock(TransactionInfo.class);
+        when(transactionInfo.getReceipt()).thenReturn(transactionReceipt);
+        when(transactionInfo.getBlockHash()).thenReturn(blockHash.getBytes());
```

Comment:
```suggestion
```
No needed.

Reply:
This is needed, if not there is a NPE here: https://github.com/rsksmart/powpeg-node/blob/10f2a458b44e64e43cdd917d9841dde1fe292db7/src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java#L81